css/nav.css

```diff
@@ -0,0 +1,6 @@
+@media (min-width: 1024px) {
+    .nav__item{
```

Comment:
Same here. If you don't have a "nav" block in the markup, you shouldn't use "nav__item", as it contradicts the BEM convention